e2e_tests/aws_lambda/test_lambda_deployment.py

```diff
@@ -25,7 +25,7 @@ def test_aws_lambda_deployment(iris_clf_service):
         '-b',
         iris_clf_service,
         '--region',
-        'us-west-2',
+        'us-east-1',
```

Comment:
no need to... it will just make our test run slower. If you want, add a unit test to cover the behavior